itests/hive-minikdc/src/test/java/org/apache/hive/minikdc/TestJdbcWithMiniKdcCookie.java

```diff
@@ -99,9 +121,11 @@ public void testCookie() throws Exception {
     }
     stmt.execute("drop table " + tableName);
     stmt.close();
+
+    testCookieNegative();
```

Comment:
Why do you execute testCookieNegative inside testCookie rather than executing as a separate test? I am not sure about the motive for this change.

Reply:
still curious for this :)